### ISSUE TITLE
basioccipital and occipital

### ISSUE THREAD

**Issue by balhoff** [obophenotype/mammalian-phenotype-ontology#2140]
I believe that 'abnormal basioccipital bone morphology' should not be a subclass of 'abnormal occipital bone morphology', since 'basioccipital bone' is not a subclass of 'occipital bone'.

Since both of these phenotypes are defined with equivalent class expressions, this relationship leads to mistaken inferences for other other expressions involving these two structures (when MP is used with other ontologies).

It looks like other subclasses of 'abnormal occipital bone morphology' might have the same problem.

**Comment by cmungall** [comment]
I believe MP is correct, as MP is scoped to mammals.

We have a taxon-GCI in uberon:

```
id: UBERON:0001692
name: basioccipital bone
relationship: part_of UBERON:0001676 {source="ISBN10:0073040584", 
gci_relation="part_of", gci_filler="NCBITaxon:40674"} ! occipital bone
```

The mistake is in assuming MP is applicable outside mammalia, as we do 
in monarch and phenoscape. Largely this is fine. Whilst many *classes* 
in MP are inapplicable for, say, fish, this is not a problem if we don't 
use those classes to describe fish. The problem arises when we use a 
class C applicable to T where C has axioms inapplicable to T.

I think this is relatively rare (should be easy to get an estimate using 
uberon taxon GCIs). It's not massively consequential for monarch, but 
for phenoscape who are interested in precise questions of loss of 
structures over entire clades this is a problem.

I think the correct fix is to have uberon-like phenotype classes that 
formally subsume MP

As a quick fix you might want to consider some kind of local override 
system (ugly I know...)

**Comment by balhoff** [comment]
Oh, I forgot about propagating phenotype over `part of`. I was focused on comparing the anatomy class hierarchy [here](http://www.informatics.jax.org/searches/AMA.cgi?id=MA:0001468#top) (and in Uberon) to the phenotype hierarchy. So considering the `part of` GCI I guess this does make sense. I am thinking of writing a little processor for MP which swaps in `X and part_of some mouse` for every anatomical term X, to use in Phenoscape.

**Comment by cindyJax** [comment]
This is a bit more complex that species differences.  The basioccipital, exoccipital and supraoccipital bones are distinct bones in prenatal, neonatal and some early juvenile stages in primates and rodents. In adult organisms, these structures are fused to form the unified adult occipital bone.  From what I understand, these are never fused in teleosts and are separate bones?
The FMA correctly has separate classes for the adult 'occipital bone' and 'region of fetal occipital bone'.  The basioccipital, exoccipital and supraoccipital bones are subclasses of the 'region of fetal occipital bone'.   I will review this for the MP and I expect that we will restructure by splitting the occipital bone term and annotations, keeping the existing term for the adult annotations, and creating another class similar to 'region of fetal occipital bone'. The basioccipital, exoccipital and supraoccipital bones can be subclasses of that new term.

**Comment by cmungall** [comment]
Great analysis thanks!

Will the proposed additional complexity buy us much in the MP case? Are there fetal abnormalities that we wouldn't want to see listed under the whole-basicoccipital grouping?

**Comment by cindyJax** [comment]
Ah- good point.  Probably this is why I organized it this way in the first place... 
I'll add some comments for clarity and perhaps revise the text definitions somewhat.

**Comment by cindyJax** [comment]
Added comments to the abnormal basioccipital, exoccipital and supraoccipital bone morphology terms for clarity.
